lib/worker.js

```diff
@@ -34,6 +34,10 @@ class Worker extends ClusterProcess {
 
         this.registerRemoteCommand(RPC.fns.worker.applyForeignProperties, this.applyForeignProperties.bind(this));
         this.registerRemoteCommand(RPC.fns.worker.broadcastMasterEvent, this.broadcastMasterEvent.bind(this));
+
+        this._suspendFunctions = [];
```

Comment:
Do we have any particular reasons to have multiple suspend functions in worker?

Imo, seems a bit like an unnecessary complication to the luster core.

Reply:
This is useful in case you have independent modules in worker. Each of them could need a suspend and coordinating them is a bit messy.